public/js/buttons.js

```diff
@@ -61,6 +61,10 @@ window.addEventListener('load', function () {
             href: 'https://girlthi.ng/~thermia/',
             src: 'https://girlthi.ng/~thermia/img/88x31/thermia.gif',
         },
+        'catwithaclarinet': {
+            href: 'https://blog.catwithaclari.net',
+            src: 'https://blog.catwithaclari.net/button.png',
+        };
```

Comment:
meow this just needs to be `},` instead of `};` :3

Reply:
oop, lol

i'll fix that real quick